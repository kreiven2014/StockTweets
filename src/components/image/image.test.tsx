/* REACT */
import React from "react";

/* MODULES */
import renderer from "react-test-renderer";

/* CUSTOM MODULES */
import Image from "./image";

describe("Image tests", () => {
  const size = 50;

  test("renders correctly", (): void => {
    const tree = renderer
      .create(
        <Image
          width={size}
          height={size}
          uri="https://via.placeholder.com/150"
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  // TODO(anybody): add test when pass image from assets !
});
