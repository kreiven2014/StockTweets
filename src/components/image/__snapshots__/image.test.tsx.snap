// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image tests renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "overflow": "hidden",
      },
      Object {
        "borderRadius": undefined,
        "height": 50,
        "width": 50,
      },
    ]
  }
>
  <FastImageView
    resizeMode="contain"
    source={
      Object {
        "uri": "https://via.placeholder.com/150",
      }
    }
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
</View>
`;
